mPemba/veryfi-challenge
# Updates Round One

## What was changed
- Updating ReadMe
- Adding simple tests
- Adding algorithm portion of challenge to UI
- Adding Sample Receipts

## How to test 
In a terminal window start the node server
`cd node-server`
`npm install`
`npm start`

In another terminal window start the app
`npm install`
`npm start`

Confirm receipt logic still works
Confirm algorithm challenge displays results in console and UI
